webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -198,52 +186,32 @@ class ManageManifestModal extends Component {
 
                     <FormGroup>
                       <Grid>
-                        <h5>{__('Subscription Manifest')}</h5>
+                        <h3>{__('Subscription Manifest')}</h3>
                         <hr />
                         <Row>
-                          <Col sm={4}>
+                          <Col sm={5}>
                             <ControlLabel
                               className="control-label"
                               htmlFor="usmaFile"
```

Comment:
This `<ControlLabel>` creates an html `<label>` element, and the `htmlFor` binds it to the form control having the ID `"usmaFile"`.  The effect is that when I click the words "Subscription Allocation," the file open dialog shows.

To fix it I think this can be moved to line 216 right before the `<FormControl>`.